web/app/view/BaseMap.js

```diff
@@ -120,9 +120,15 @@ Ext.define('Traccar.view.BaseMap', {
         });
 
         this.map.on('click', function (e) {
-            this.map.forEachFeatureAtPixel(e.pixel, function (feature, layer) {
-                this.fireEvent('selectfeature', feature);
-            }.bind(this));
+            if (this.map.hasFeatureAtPixel(e.pixel, { layerFilter: function (layer) {
```

Comment:
It's still not correct. Everything after curly brace should be on a new line. Also there should be no 2 curly braces on the same line at the end. Split it into different lines.